src/shell/js/composeui-node-launcher/tsconfig.json

```diff
@@ -9,7 +9,7 @@
     "declaration": true,
     "outDir": "output",
     "lib": [ "es2022", "DOM" ],
-    "strict": true
+    "strict": false
```

Comment:
Uhm, are we sure about this?

Reply:
we can't use 'with' in strict mode.

Reply:
ok, I did a double check. It looks like that fortunately I only need to remove the enforcement of the strict mode from the script file, but not from the tsconfig.json files. I've reset the value to true.